olcPGEX_AdditionalColours.h

```diff
@@ -6,13 +6,81 @@ namespace olc
 {
 	namespace Colours
 	{
+		//ALL THE COLORS OF THE WIND!
 		static const Pixel
+			BLACK(24, 24, 26),
```

Comment:
I don't think we should be overriding the colours that are already defined in the PGE as that could be confusing. For example, I think the `BLACK` and `WHITE` ones here would be better named something like `OFF_BLACK` and `OFF_WHITE`